src/sources/docker_logs.rs

```diff
@@ -1365,16 +1365,18 @@ mod integration_tests {
         trace_init();
 
         let will_be_read = "12";
-        let all_containers = "vector_test";
-        let excluded0 = "vector_test_exclude_0";
-        let included0 = "vector_test_include_0";
-        let included1 = "vector_test_include_1";
+        let include_prefix = "include_me_plz";
+        let included0 = "include_me_plz_0";
+        let included1 = "include_me_plz_1";
+
+        let exclude_prefix = "exclude_me_plz";
```

Comment:
I think this will result in the same behavior I mentioned earlier where even if `exclude_containers` wasn't specified, it would pass due to the `include_prefix`. I might suggest something like:

```rust
let prefix = "vector_test_exclude_containers"; // this matches the naming conventions from other tests in this file
let included0 = format!("{}_{}", prefix, "include0");
let included1 = format!("{}_{}", prefix, "include1");
let excluded0 = format!("{}_{}", prefix, "exclude0");
```

and then `include_containers: Some(vec![prefix.to_owned()])` and `exclude_containers: Some(vec![exclude0.to_owned()])`.

This should test the behavior that `exclude0` would have been included if `exclude_containers` was missing.